### PR TITLE
blog: Add "Release candidate: Godot 4.0 RC 2"

### PR DESCRIPTION
**Pull request by akien-mga**

### CODE REVIEW

File: collections/_article/release-candidate-godot-4-0-rc-2.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+- Rendering: Vulkan: Make draw command labels thread safe ([GH-73057](https://github.com/godotengine/godot/pull/73057)).
+- Rendering: OpenGL: Store blend mode between CanvasItems to preserve batching ([GH-73255](https://github.com/godotengine/godot/pull/73255)).
+- Rendering: Notify mesh surface when `render_priority` changes ([GH-73263](https://github.com/godotengine/godot/pull/73263)).
+- Rendering: Bias octahedral tangent y axis to avoid errors around 0 ([GH-73265](https://github.com/godotengine/godot/pull/73265)).
+- Shaders: Avoid inserting `screen_texture` compatibility code when using `textureSize` ([GH-73060](https://github.com/godotengine/godot/pull/73060)).
+- Thirdparty: Update libwebp to upstream 1.3.0 ([GH-72045](https://github.com/godotengine/godot/pull/72045)).
+
+This release is built from commit [d2699dc7a](https://github.com/godotengine/godot/commit/d2699dc7ab96fbd75faccc1f32f55baebf1d84dc).
+
+<a id="downloads"></a>
```
Comment (coppolaemilio) [inline collections/_article/release-candidate-godot-4-0-rc-2.md]: Never caught this one but I think this should be replaced with a `div` or other tag. `<a>` tags should always have an `href` value.
```suggestion
<div id="downloads"></div>
```

Comment (akien-mga) [reply collections/_article/release-candidate-godot-4-0-rc-2.md]: Fixed!

File: collections/_article/release-candidate-godot-4-0-rc-2.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+The Production team has been hard at work assessing the remaining PRs and issues in the 4.0 milestone, pushing non-critical or risky changes to the 4.1 milestone, or prioritizing what could actually still be done so close to the release. Astute watchers of the [4.0 milestone](https://github.com/godotengine/godot/milestone/9)'s completion ratio will have noticed a significant increase, but that doesn't mean that all issues are being solved. Some will be fixed in time for 4.1, and cherry-picked for 4.0.x maintenance releases along the way.
+
+Regardless of which milestone our contributors assign to your issues, be sure to keep reporting any major or minor problem which you're having with 4.0 RC 2! Your test reports at this stage are critical for us to assess whether we can finally stamp the current development branch as "stable" and release it to all platforms.
```
Comment (coppolaemilio) [inline collections/_article/release-candidate-godot-4-0-rc-2.md]:
```suggestion
Regardless of which milestone our contributors assign to your issues, be sure to keep reporting any major or minor problem you're having with 4.0 RC 2! Your test reports at this stage are critical for us to assess whether we can finally stamp the current development branch as "stable" and release it to all platforms.
```

**Review by YuriSizov — APPROVED** [review]
LGTM aside from the cover image missing the overlay :P

**Comment by akien-mga** [comment]
> LGTM aside from the cover image missing the overlay :P

Thanks, fixed :)

**Pull request merged**